webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -18,7 +19,23 @@
           :src="contribution.image | proxyApiUrl"
         />
       </hc-teaser-image>
+
       <ds-card>
+        <div class="blur-toggle">
+          <label for="blur-img">{{ $t('contribution.inappropriatePicture') }}</label>
+          <input type="checkbox" id="blur-img" v-model="form.blurImage" />
+          <p>
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-12-04T19:03:43Z" title="Wednesday, December 4th 2019, 8:03:43 pm +01:00">Dec 4, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -7,14 +7,57 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
-      <hc-teaser-image :contribution="contribution" @addTeaserImage="addTeaserImage">
+      <hc-teaser-image
+        :contribution="contribution"
+        @addTeaserImage="addTeaserImage"
+        :class="{ 'images-set-blur': checkedBlur }"
+      >
         <img
           v-if="contribution"
           class="contribution-image"
           :src="contribution.image | proxyApiUrl"
         />
       </hc-teaser-image>
+      <ds-text align="right" class="blurBox">
+        <div v-show="checkedBlur">
```


```suggestion
        <div v-show="blurImage">
```